src/app/components/SheriffLocationDetails.tsx

```diff
@@ -1,16 +1,20 @@
 import * as React from 'react';
-import { Sheriff } from '../api/index';
 import {
     Table
 } from 'react-bootstrap';
+import CourthouseCard from './CourthouseCard';
+import RegionCard from './RegionCard';
 
-export interface TrainingDetailsViewProps {
-    sheriff: Sheriff;
+export interface SheriffLocationDetailsProps {
+    currentCourthouse?: string,
```

Comment:
These are Id's now? perhaps **currentCourthouseId** . Another option might be that we combine the two into an object called a **Location** i.e.:
```ts
interface SheriffLocation{
   courthouseId:string;
   regionId: string;
}

export interface SheriffLocationDetailsProps{
   currentLocation?: SheriffLocation
   permanentLocation?: SheriffLocation
}

```
This would likely be something we'd want to put into the API.
Just an idea.

Reply:
I like the idea of creating a SheriffLocation in the API - I will add it in